### ISSUE TITLE
4.0 Field Layout editor slideouts should include a “Cancel” option

### ISSUE THREAD

**Issue by mattstein** [craftcms/cms#10587]
### Description

Field Layout editor slideouts can be dismissed via the <kbd>esc</kbd> key and by clicking outside the slideout, but for UX clarity and accessibility there should be an explicit option to cancel or dismiss changes so it’s not left to the user to guess.

![Screen Shot 2022-02-16 at 11 30 47 AM@2x](https://user-images.githubusercontent.com/2488775/154341361-1b7c1e67-2635-4b7b-820d-12f474400871.png)

### Steps to reproduce

1. Pick an entry type to edit. (**Settings** → **Sections** → [any **Edit entry type**].)
2. Click the gear icon next to any field in the **Field Layout** editor.
3. Make some edits in the slideout you decide you don’t want to apply, then find the control that allows you to discard them.

### Additional info

- Craft version: 4.0.x-dev (decee1422ca2b93efc3d0ebec93c4eff04523111)

**Comment by brandonkelly** [comment]
Added a “Close” button to them for the next release.

(Un-applied changes are not actually lost when the field layout designer slideouts are closed, so “Cancel” wouldn’t have been appropriate as it implies the changes will be discarded.)

**Comment by mattstein** [comment]
Makes sense, thank you!

**Comment by brandonkelly** [comment]
Craft 4.0.0-beta.4 is out now with that change.
